campgns/twinkprs/map00203.txt

```diff
@@ -14,6 +14,9 @@ REM    the Free Software Foundation; either version 2 of the License, or
 REM    (at your option) any later version.
 REM ****************************************************************************
 
+LEVEL_VERSION(1)
+SET_GAME_RULE(BREAK_NEUTRAL_WALLS,64)
```

Comment:
This is an incorrect level script command. It would work like this.

```
REM BREAK_NEUTRAL_WALLS
SET_GAME_RULE(PreserveClassicBugs,64)
```

Reply:
Checking though, it seems the change is completely unwanted. I provided feedback on the issue.